actions/commands.go

```diff
@@ -135,6 +135,22 @@ func Commands() {
 				return nil
 			},
 		},
+
+		{
+			Name:    "templates",
+			Usage:   "Manage project templates",
+			Subcommands: []cli.Command{
+				{
+					Name:  "list",
```

Comment:
Are we going to have more sub commands for `templates`? If not, i would maybe have this as a flag?

Reply:
we will :)